src/main/java/file/Saver.java

```diff
@@ -36,8 +36,10 @@ private static String constructJson(PlayerProfile playerProfile) {
         return "{\n" +
                 " \"name\": \"" + playerProfile.getName() + "\", \n" +
                 " \"occupation\": \"" + playerProfile.getOccupation() + "\", \n" +
-                " \"asset\": \"" + playerProfile.getAsset().outputAsset() + "\", \n" +
-                " \"health\": \"" + playerProfile.getHealth().outputHealth() + "\", \n" +
+                " \"asset\": \"" + playerProfile.getAsset() + "\", \n" +
+                " \"health\": \"" + playerProfile.getHealth() + "\", \n" +
+                " \"currentRound\": \"" + playerProfile.getCurrentRound() + "\", \n" +
+                " \"isAdvancedPlayer\": \"" + playerProfile.isAdvancedPlayer() + "\" \n" +
                 "}";
```

Comment:
How can I store an attribute which is a class?

Reply:
just convert it into string, we can define a method to do this